libs/ledgerjs/packages/hw-app-helium/package.json

```diff
@@ -29,16 +29,17 @@
   "dependencies": {
     "@helium/address": "^4.0.0",
     "@helium/proto": "^1.5.0",
-    "@helium/transactions": "^3.62.0",
+    "@helium/transactions": "^4.1.0",
     "@ledgerhq/errors": "workspace:^",
     "@ledgerhq/hw-transport": "workspace:^",
     "@ledgerhq/logs": "workspace:^",
     "bignumber.js": "^9.0.1",
-    "bip32-path": "^0.4.2"
+    "bip32-path": "^0.4.2",
+    "long": "^5.2.0"
```

Comment:
Hi @matthewcarlreetz is this library safe in React Native environment?

Reply:
Yes, I have tested it using React Native